admin/src/pages/Add.jsx

```
@@ -20,7 +20,54 @@
+    }
+  };
+  const handleUpload = async () => {
+    if (!file) {
+      return toast.error("No file Selected");
+    }
+    const formData = new FormData();
+    formData.append("file", file);
+
+    try {
+      const res = await axios.post(
+        backendUrl + "/api/product/uploadBulkProduct",
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)